yarn.lock

```diff
@@ -0,0 +1,9639 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
`yarn` is not used in the docs repo, but `pnpm` instead. `yarn.lock` file should be removed.

Reply:
ah yeah, sorry 👍🏼